web-ui/src/pages/ViewFeedbackPage.jsx

```diff
@@ -399,15 +429,15 @@ const ViewFeedbackPage = () => {
             </TextField>
           </FormControl>
 
-          <FormControl className={classes.textField} value={sortValue}>
+          <FormControl className={classes.textField}>
             <TextField
               id="select-sort-method"
               select
               fullWidth
               size="small"
               label="Sort by"
               onChange={e => setSortValue(e.target.value)}
-              defaultValue={SortOption.SENT_DATE}
```

Comment:
We don't want to always default to SENT_DATE, but instead want to use the value in the corresponding query parameter.